torch/_subclasses/fake_tensor.py

```diff
@@ -947,6 +950,9 @@ def _cache_key(
         if func in self.lift_fns:
             raise _BypassDispatchCache("lift")
 
+        if func.name() == "inductor::resize_storage_bytes_":
+            raise _BypassDispatchCache("inductor::resize_storage_bytes_")
```

Comment:
I didn't actually test that we need this skip, but it feels like the right thing to do: handling fake tensor caching on metadata mutations is complicated, and we already ban caching for inplace views.